### PR TITLE
Disclaimer

### PR DESCRIPTION
**Pull request by makoto**
I was initially thinking about writing more detailed condition but now thinks this is good enough for now.

### CODE REVIEW

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr58-app-wearekickback.surge.sh

File: src/routes/Terms.js [hunk 1]
```diff
@@ -90,6 +90,9 @@ Event attendance is determined by the event organisers or an event admin.
                 <li>
                     If the event organiser did not check you in, you will lose any funds you committed. These funds will be divided among the event attendees or they can be applied in other ways (charitable contribution, sponsorship of future events etc)
                 </li>
+                <li>
+                    We are not liable to any disputes about your attendance status. It is the event organizer's discretion to decide whether you are considered attended to the event or not.
```
Comment (hiddentao) [inline src/routes/Terms.js]:
```suggestion
We are not liable for any losses resulting from for any disputes regarding your attendance status. Whether you are marked as attended or not is strictly at the event organizers' discretion.
```

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr58-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr58-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr58-app-wearekickback.surge.sh

**Pull request merged**